.ebextensions/04_create_apache_conf.config

```diff
@@ -0,0 +1,30 @@
+#
+# Create an Apache .conf file for the preview service.
+#
+files:
+  '/tmp/suitec-preview-service.httpd.conf' :
+    mode: '000755'
```

Comment:
@johncrossman Yes, good catch; that must have been copy/paste from some other config.